Update README: meet.jit.si no longer anonymous (#13797)
jitsi/jitsi-meet: README.md

## Using Jitsi Meet

> Using Jitsi Meet is straightforward, as it's browser based. Head over to [meet.jit.si](https://meet.jit.si) and give it a try. It's scalable and free to use. It's no longer anonymous, now requiring either a Google, a Facebook or a Github login in order to start a meeting. All browsers are supported!

Reviewer: ```suggestion
Using Jitsi Meet is straightforward, as it's browser based. Head over to [meet.jit.si](https://meet.jit.si) and give it a try. It's scalable and free to use. All you need is a Google, a Facebook or a GitHub account in order to start a meeting. All browsers are supported!
```
